tools/cpp/runfiles/runfiles_src.h

```diff
@@ -30,15 +30,20 @@
 //       using bazel::tools::cpp::runfiles::Runfiles;
 //
 // 3.  Create a Runfiles object and use rlocation to look up runfile paths:
+//     Note that if Runfiles::CurrentRepository() is not supported for the cc
```

Comment:
It could make sense to mention that C++20 is a requirement (if that's actually true) to give users an idea when they should expect this to be available.